helpers/validator.js

```diff
@@ -0,0 +1,19 @@
+const Joi = require('joi');
+
+function validateInfo(inVal) {
+	const schema = {
+		name: Joi.string()
+			.min(2)
+			.required(),
+		photo: Joi.string()
+			.min(6)
+			.required(),
+		likes: Joi.number()
+			.min(1)
+			.required(),
+	};
+
+	return Joi.validate(inVal, schema);
+}
+
+module.exports = validateInfo
```

Comment:
it's usually better to return an object rather than a function itself, that way you can return a set of functions, so it's more flexible: 
```
function validateUser....

module.exports = {
 validateUser
}
```

@ScottRhodie also observe how I renamed it to 'validateUser'. It's generally a bad idea to name things like 'data', 'info' etc as they are too generic

Reply:
Hey,  

I went ahead and made these changes.  Look a little but it sort of clicked now how we're calling specific functions from other files. Name has been made less generic now also.  Would it be more suitable to have a folder called "Functions" or something like that, and have the functions held there instead of helpers?